src/main/groovy/de/itemis/mps/gradle/CreateDmg.groovy

```diff
@@ -59,11 +73,19 @@ class CreateDmg extends DefaultTask {
                             'Mac/Finder/DSStore/BuddyAllocator.pm', 'Mac/Finder/DSStore.pm']
         File scriptsDir = File.createTempDir()
         File dmgDir = File.createTempDir()
+        def getSigningInfo = [signKeyChainPassword, signKeyChain, signIdentity]
```

Comment:
The variable should just be called `signingInfo`, the `get` prefix is confusing.